proxy/context.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/zalando/skipper/routing"
 )
 
+const unknownHost = "_unknownhost_"
```

Comment:
consistency. you have `_unknown_method_` and `_unknownhost_`. I would make it consistent and change to `_unknown_host_` - also reads nicer